torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -1440,10 +1440,12 @@ void ProcessGroupNCCL::watchdogHandler() {
     // Bump up heart beat by one.
     heartbeat_++;
 
-    // poll store to see if some ranks have flagged a timeout when
+    // Assuming that we always init a process group containing all ranks,
+    // we only use the watchdog thread for that PG to detect timeout dump.
```

Comment:
> to detect timeout dump

change to
"to listen for the global signal to dump and abort"

because we still are using every thread for locally detecting timeout/dump, and this might confuse ppl

Reply:
ok..